x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -161,8 +161,12 @@ const validateFormValues = (values: MaybeImmutable<NewTrustedApp>): ValidationRe
       } else if (
         !isPathValid({ os: values.os, field: entry.field, type: entry.type, value: entry.value })
       ) {
-        // show soft warnings and thus allow entry
-        isValid = true;
+        if (!values.name.trim()) {
```

Comment:
Name is already validated at the beginning (line 100). Instead of checking the name again, can you just check if the `isValid` is true or false? This will allow us to rely just on isValid for next validations and avoid having the same issue in the future with other fields.

Reply:
Ah yes! There's no need to set `isValid` here at all then.

Reply:
done 121258cc16edbe420a7afcdf32664527ed6b68bd